app/Models/Permission.php

```diff
@@ -16,7 +16,8 @@ class Permission extends Model
      * Constants defining different permissions available.
      */
     public const ACTION_WEBSOCKET_CONNECT = 'websocket.connect';
-    public const ACTION_CONTROL_CONSOLE = 'control.console';
+    public const ACTION_CONTROL_CONSOLE_SEND = 'control.sendconsole';
```

Comment:
Use dashes when multiple words are being used rather than putting them together.  `control.send-console` for example.